README.md

```diff
@@ -1,27 +1,16 @@
 # NServiceBus.AmazonSQS
 
-This is an Amazon SQS transport for NServiceBus.
+NServiceBus.AmazonSQS is the Amazon SQS transport for NServiceBus.
 
-Feel free to browse and contribute!
+It is part of the [Particular Service Platform](https://particular.net/service-platform), which includes [NServiceBus](https://particular.net/nservicebus) and tools to build, monitor, and debug distributed systems.
 
-For more information, including a guide on getting started quickly, see the project documentation at [docs.particular.net](https://docs.particular.net/transports/sqs/).
+See the [Amazon SQS Transport documentation](https://docs.particular.net/transports/sqs/) to learn more.
 
-## Running the Acceptance Tests
+## Running tests locally
 
 The solution contains the [NServiceBus Acceptance Test suite](https://www.nuget.org/packages/NServiceBus.AcceptanceTests.Sources/) and the [NServiceBus Transport Test suite](https://www.nuget.org/packages/NServiceBus.TransportTests.Sources/).
-To run the tests, the Access Key ID and Secret Access Key of an AWS IAM account need to be set in environment variables on the machine running the tests. Full details on how to set this up can be found [here](https://docs.particular.net/transports/sqs/#getting-started-set-up-an-aws-account).
 
-The transport can be configured using the following environment variables:
-
-* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#s3bucketforlargemessages) parameter. Default is no S3 bucket.
-
-The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required.
```

Comment:
I think it is best to add this back in as the variable is used in ci.yml 
* **NSERVICEBUS_AMAZONSQS_S3BUCKET** corresponds to the [S3BucketForLargeMessages](https://docs.particular.net/transports/sqs/configuration-options#offload-large-messages-to-s3) parameter. By default this is disabled. The bucket should not have encryption enabled. An additional bucket `{NSERVICEBUS_AMAZONSQS_S3BUCKET}.kms` with AWS KMS encryption enabled is required. For further details review the documentation on [configuration options](https://docs.particular.net/transports/sqs/configuration-options).